_includes/glossary/README.md

```diff
@@ -0,0 +1,38 @@
+# Kubernetes Glossary
+
+To write a glossary snippet, start with a copy of the template in this directory, `_glossary-template.md`. Make sure to provide (or omit) values for the following fields:
+
+* (Required) `term`
+* (Optional) `formerly` and `related`. If you do not provide these values, remove the fields.
+* (Required) short description. Make sure to replace the instructional text in the template with your content.
+* (Optional) long description. If you do not provide a long description, remove the `{% capture %}` and `{% endcapture %}` tags together with the instructional text.
+
+The template also contains basic information about how to write your snippet. For additional guidance, read on in this readme.
+
+## Glossary snippet style guide
+
+This style guide supplements the guidance provided in the glossary template. It's intended to help you think about what and how to write glossary definitions. For more general guidance on style, consult [the core docs style guide](https://kubernetes.io/docs/home/contribute/style-guide/), also available in this repo at `docs/home/contribute/style-guide.md`.
```

Comment:
I added it because I am constantly hunting for it myself when I have the repo open in my local editor. And I don't like linking out from a readme in the repo (if I'm viewing in the browser) to the built docs. So this was my compromise. Which was only a thought. Happy to remove (standard practice seems to be to link only to the built HTML).